lib/rules/no-use-computed-property-like-method.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * @author tyankatsu <https://github.com/tyankatsu0105>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+const eslitUtils = require('eslint-utils')
+const utils = require('../utils')
+
+/**
+ * @typedef {import('../utils').ComponentPropertyData} ComponentPropertyData
+ * @typedef {import('../utils').ComponentObjectPropertyData} ComponentObjectPropertyData
+ *
+ * @typedef {{[key: string]: ComponentPropertyData & { valueType: { type: string } }}} ProprtyMap
+ */
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/**
+ *
+ * @param {ComponentObjectPropertyData} property
+ * @return {string | null}
+ */
+const getComponetPropsType = (property) => {
+  if (property.property.value.type === 'ObjectExpression') {
+    const propsTypeProperty = property.property.value.properties.find(
```

Comment:
Perhaps you can use `utils.findProperty`.

Reply:
Wow. The findProperty looks awesome function!
Thanks.
https://github.com/vuejs/eslint-plugin-vue/pull/1234/commits/d885a3633a4eb6f56dbe8d3262ac66b0fe6967cf